features/no-disk-space.feature

```diff
@@ -0,0 +1,16 @@
+Feature: No Disk Space Overlay
+
+  Background:
+    Given I have completed the basic setup
+
+  Scenario: I set the required space to 1 KB
```

Comment:
@daniloprates the scenario title should be more descriptive - e.g. "No disk space overlay is not shown when there is enough available disk space"